tox.ini

```diff
@@ -2,11 +2,12 @@
 skip_missing_interpreters = True
 minversion = 1.8
 envlist =
-	py{py,27,36}-requests,style,isort-check
+	py{py,27,36}-requests,style,isort-check,integration-tests
 
 [testenv]
 commands =
-    py.test {posargs:-vvs}
+    py.test {posargs:-vvs -k "not test_integration"}
```

Comment:
probably want to use e.g. `--ignore=tests/test_integration.py` form here vs `-k`.

I think that should also go outside of the posargs default, too. otherwise passing any kind of positional args will override this and cause integration tests to run where they would not normally.

Reply:
I used the default for the purpose of overriding with a positional arg. The issue is that `py.test --ignore=tests/test_integration.py -k tests/test_integration.py -vvs` results in no tests being run. Any ideas?

Reply:
> The issue is that py.test --ignore=tests/test_integration.py -k tests/test_integration.py -vvs results in no tests being run. 

ah, I see - because the `integration-tests` env itself relies on this for its own execution. `-k not tests/test_integration.py` sounds good, but probably outside of the `posargs`?